src/main/telemetry/smartport.c

```diff
@@ -126,6 +126,8 @@ enum
     FSSP_DATAID_CELLS_LAST = 0x030F ,
     FSSP_DATAID_HEADING    = 0x0840 ,
 #if defined(USE_ACC)
+    FSSP_DATAID_PITCH      = 0x0430 ,
+    FSSP_DATAID_ROLL       = 0x0440 ,
```

Comment:
These are wrong according to the SmartPort specification.

Reply:
Afaict, smartport has no id for such a sensor, the id chosen is not reserved for any existing sensor, and it's consistent with inav. Given it is what the majority of software expects, these ids doesn't seem unreasonable. It is not dissimilar from the usage of FSSP_DATAID_T1. I can add a cli setting for this usage of these ids, if that would make it more acceptable.

Reply:
Using a sensor id that has not yet been assigned is wrong, as these sensor ids may be used by FrSky for some other sensor in the future without consultation. This has led to conflicts in the past.
So can I suggest that you either:
- apply to FrSky to have these sensor ids added to the SmartPort specification; or
- use ids out of the range reserved for non-official use (0x5100 to 0x52ff) instead?

Reply:
I went for the 2nd option as this satisfies the requirements for my use case.